frontend/packages/dev-console/src/components/helm/helm-utils.ts

```diff
@@ -126,3 +128,12 @@ export const flattenReleaseResources = (resources: { [kind: string]: { data: K8s
     }
     return acc;
   }, []);
+
+export const getChartValuesYAML = (chart: HelmChart): string => {
+  const orderedValuesFile = chart?.files.find((file) => file.name === 'ordered-values.yaml');
```

Comment:
will chart always have `files` if not then maybe we can have `chart?.files?.find`

Reply:
Chart data is supposed to always have files. But it doesn't hurt anyone to add extra check.

Reply:
Updated.